includes/class-kadence-blocks-prebuilt-library-rest-api.php

```diff
@@ -600,6 +607,46 @@ public function get_remaining_credits( $request ) {
 			return rest_ensure_response( $response );
 		}
 	}
+	/**
+	 * Handle the event and maybe send to Prophecy WP.
+	 */
+	public function handle_event( $request ) {
+		$event_label = $request->get_param( self::PROP_EVENT_LABEL );
+		$event_data  = $request->get_param( self::PROP_EVENT_DATA );
+		$event_data  = json_decode( $event_data, true );
+
+		$event       = '';
+		$context     = array();
+
+		switch ( $event_label ) {
+			case 'ai_wizard_started':
+				$event = 'ai_wizard_started';
```

Comment:
Thinking through this a little more, everything I've seen in [Mixpanel's documentation](https://docs.mixpanel.com/docs/data-structure/events-and-properties#name-events-and-properties-consistently) suggests event names like "AI Wizard Started" so let's be consistent with the event names. You're `event_label` here is fine to collect the necessary event data.